WalletWasabi/Crypto/ZeroKnowledge/Transcript.cs

```diff
@@ -58,7 +58,7 @@ public void CommitPublicNonces(IEnumerable<GroupElement> publicNonces)
 			AddMessages(PublicNonceTag, publicNonces.Select(x => x.ToBytes()));
 		}
 
-		public void CommitStatement(Statement statement)
+		public void CommitStatement(IStatementDescription statement)
 		{
 			CryptoGuard.NotNullOrInfinity(nameof(statement.Generators), statement.Generators);
 			AddMessages(StatementTag, statement.Generators.Select(x => x.ToBytes()));
```

Comment:
Should not we also commit here to the Public points used in the statement? Or is it handled somewhere else?

Reply:
good catch, this method was rewritten so many times i got sloppy